src/player.cpp

```diff
@@ -11302,10 +11299,6 @@ void player::do_read( item *book )
         add_msg(m_info, _("You can no longer learn from %s."), book->type_name().c_str());
     }
 
-    for( auto &m : reading->use_methods ) {
```

Comment:
Do the martial art magazines still work, they seem to use the same technique to call functions as the note item.

This seems to remove the only usage of `islot_book::use_methods`. If it's indeed not used anywhere, it should be removed from the class itself.